### PR TITLE
Fix some alerts that were detected by the LGTM system

### PR DESCRIPTION
**Pull request by tdrozdovsky**
Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

Previously, the nnstream project was analyzed by the LGTM system. [Alerts were discovered](https://lgtm.com/projects/g/nnstreamer/nnstreamer/), some of which are fixed in this PR. 

**Self evaluation:**
1. Build test: [x] Passed [ ]Failed []Skipped
2. Run test: [x] Passed [ ]Failed []Skipped

### CODE REVIEW

**Review by jaeyun-jung — APPROVED** [review]

File: gst/nnstreamer/tensor_converter/tensor_converter.c [hunk 1]
```diff
@@ -1037,7 +1037,7 @@ gst_tensor_converter_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
       type = gst_tensor_get_element_size (config->info.info[0].type);
 
       /** colorspace * width * height * type */
-      frame_size = color * width * height * type;
+      frame_size = (gsize) color *width * height * type;
```
Comment (gichan-jang) [inline gst/nnstreamer/tensor_converter/tensor_converter.c:1040]: minor things.
```suggestion
      frame_size = (gsize) color * width * height * type;
```
L1068 also.

Comment (tdrozdovsky) [reply gst/nnstreamer/tensor_converter/tensor_converter.c:1040]: For me, this was also a surprise. The indent-format system issued me a mistake and the following patch was proposed.
```
@@ -1037,7 +1037,7 @@ gst_tensor_converter_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
       type = gst_tensor_get_element_size (config->info.info[0].type);
 
       /** colorspace * width * height * type */
-      frame_size = (gsize) color * width * height * type;
+      frame_size = (gsize) color *width * height * type;
 
       /** supposed 1 frame in buffer */
       g_assert ((buf_size / self->frame_size) == 1);
@@ -1065,7 +1065,7 @@ gst_tensor_converter_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
         /**
          * Refer: https://gstreamer.freedesktop.org/documentation/design/mediatype-video-raw.html
          */
-        size = offset = (size_t) color * width * type;
+        size = offset = (size_t) color *width * type;
 
         g_assert (offset % 4); /** Internal logic error! */
         if (offset % 4) {

```
And I was forced to do so.

Comment (gichan-jang) [reply gst/nnstreamer/tensor_converter/tensor_converter.c:1040]: It seems to be a problem with the LGTM system.
You should fix the system. :)
Please modify this regardless of the system.

Comment (tdrozdovsky) [reply gst/nnstreamer/tensor_converter/tensor_converter.c:1040]: No, there is error from TAOS-Ci (indent-format).

Comment (tdrozdovsky) [reply gst/nnstreamer/tensor_converter/tensor_converter.c:1040]: LGTM does not check the format code.

Comment (tdrozdovsky) [reply gst/nnstreamer/tensor_converter/tensor_converter.c:1040]: Of course I can try to add a space. If you say you need!

Comment (gichan-jang) [reply gst/nnstreamer/tensor_converter/tensor_converter.c]: Oh.. You're right. TAOS-CI detected it.
Let's find the problem later.

Comment (gichan-jang) [reply gst/nnstreamer/tensor_converter/tensor_converter.c]: I checked it on my local PC. Thanks. Please revert your last change.

Comment (tdrozdovsky) [reply gst/nnstreamer/tensor_converter/tensor_converter.c]: Will I revert?

Comment (gichan-jang) [reply gst/nnstreamer/tensor_converter/tensor_converter.c]: Yes. please.

Comment (tdrozdovsky) [reply gst/nnstreamer/tensor_converter/tensor_converter.c]: One moment!

Comment (tdrozdovsky) [reply gst/nnstreamer/tensor_converter/tensor_converter.c:1040]: Done!

Comment (gichan-jang) [reply gst/nnstreamer/tensor_converter/tensor_converter.c:1040]: @tdrozdovsky How about this as temporal solution for weird indent.
```
frame_size = (gsize) (color * width * height * type);
```

Comment (tdrozdovsky) [reply gst/nnstreamer/tensor_converter/tensor_converter.c]: @gichan-jang  Done!

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by myungjoo — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**